.gitignore

```diff
@@ -0,0 +1,19 @@
+#some of these may still be checked in; also using this
+#to ignore files when zipping repo for data release
+*.ipynb
+run_lstm_check_results.ipynb
```

Comment:
Is this supposed to have a `!` in front of it? I see this file committed in this PR

Reply:
The .gitignore entry is just to ignore it in `copy_zip_repo` so it stays out of the data release